external-resolver/src/test/scala/org/genivi/sota/resolver/test/random/Query.scala

```diff
@@ -162,7 +162,7 @@ object Query extends
           Store.pickPackageWithFilter.runA(s) map { case (pkg, flt) => ListFiltersFor(pkg)  }
         )),
 
-        (if (pkgs > 0) 50 else 0,
+        (if (pkgs > 0) 10 else 0,
```

Comment:
Previously half of all queries were `Resolve` making the other kinds so uncommon that the `Random` test took roughly twice longer to attain full coverage.